apps/summercountdown/summer_countdown.star

```diff
@@ -0,0 +1,482 @@
+                            child = render.Circle(color = COLOR_SEEDS, diameter = int(huge_flower_diameter * 0.541)),
+                        ),
+                    ),
+                )
+
+            if huge_flower_growth_done_frame == None and huge_flower_diameter == HUGE_FLOWER_MAX_DIAMETER:
+                huge_flower_growth_done_frame = i
+
+        if huge_flower_growth_done_frame and i >= huge_flower_growth_done_frame + 4:
+            # Display the countdown message:
+            if 0 < days_until and days_until < 10:
+                days_until_text = " " + str(days_until)
+            else:
+                days_until_text = str(days_until)
+
+            # Set up widgets for 2 or 3 digit countdowns
+            if len(days_until_text) > 2:
+                font = "6x13"
+                digits_pad = (8, 2, 0, 0)
+                msg_pad = (7, 0, 0, 0)
+                plural_cover_pad = (41, 6, 0, 0)
+            else:
+                font = "10x20"
+                digits_pad = (6, -1, 0, 0)
+                msg_pad = (7, 2, 0, 0)
+                plural_cover_pad = (41, 8, 0, 0)
+
+            # Show the digits and message
+            frame_stack.append(
+                render.Stack(children = [
+                    render.Padding(
+                        pad = digits_pad,
+                        child = render.Text(
+                            content = days_until_text,
+                            font = font,
+                            color = COLOR_MESSAGE,
+                        ),
+                    ),
+                    render.Padding(
+                        pad = msg_pad,
+                        child = render.Image(src = base64.decode(countdown_png)),
+                    ),
+                ]),
+            )
+
+            # Cover up the "S" if days_until is singular
+            if days_until == 1:
+                frame_stack.append(
+                    render.Padding(
+                        pad = plural_cover_pad,
+                        child = render.Box(height = 7, width = 4, color = COLOR_SEEDS),
+                    ),
+                )
+
+            # Ripen sunflower seeds to reveal the countdown
+            frame_stack.append(render.Stack(children = huge_seed_widgets))
+            ripen_random_seeds(50, huge_seed_widgets, 0.2)
+
+        frames.append(render.Stack(children = frame_stack))
+    return frames
+
+def main(config):
+    frames = render_all_frames(FRAME_COUNT, config)
+    return render.Root(
+        child = render.Animation(children = frames),
+        delay = int(1000 / FPS),
+    )
+
+def get_schema():
+    hemispheres = [
+        schema.Option(
+            display = "Northern Hemisphere",
+            value = "northern",
+        ),
+        schema.Option(
+            display = "Southern Hemisphere",
+            value = "southern",
+        ),
+    ]
+
+    return schema.Schema(
+        version = "1",
+        fields = [
+            schema.Dropdown(
```

Comment:
If you used a `schema.Location`, it would be auto-populated from the device location and you could determine from the latitude :D